### PR TITLE
Population trend graph

### PR DESCRIPTION
**Pull request by tinashechiraya**
Description:

I have added a line chart similar to the one on the dashboard
It's configured to display points (dots) with a specified size and color.
With this the chart has dots representing the actual counts per year on the chart,
in addition to the existing trend line and confidence intervals.

On local host I don't have the permissions to get the data but the implementation is the same as the charts on the dashboard 
image

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 55 sec

File: django_project/frontend/models/statistical.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 PROVINCE_TREND = 'province_trend'
 PROPERTY_TREND = 'property_trend'
 POPULATION_PER_PROPERTY = 'population_per_property'
+SPECIE_PER_PROPERTY = 'specie_per_property'
```
Comment (dimasciput) [inline django_project/frontend/models/statistical.py:13]: Species is both singular and plural

File: django_project/frontend/api_views/statistical.py [hunk 2]
```diff
@@ -78,3 +80,100 @@ def get(self, *args, **kwargs):
...
+        rows = AnnualPopulation.objects.select_related(
+            'owned_species', 'survey_method',
+            'owned_species__taxon',
+            'owned_species__property', 'owned_species__property__province',
+            'owned_species__property__property_type'
+        ).filter(
+            query
+        ).order_by('year')
+
+        csv_data = [
```
Comment (dimasciput) [inline django_project/frontend/api_views/statistical.py:151]: please use serializer instead

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 11 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 7 min 11 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 5 min 6 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 4 min 41 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 4 min 54 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 4 min 51 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 52 sec

**Comment by tinashechiraya** [comment]
@dimasciput I have applied the changes

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 46 sec

**Pull request merged**